### DISCUSSION TITLE
try install two argocd in different namespace

### DISCUSSION THREAD

**Discussion by BinaryWorld2099** [argoproj/argo-cd#10753] [Q&A]
I was trying to install another new argocd in another namespace. but get an error

the existing old one was working fine in argocd namespace (installed with helm) , but using helm again trying to install another one in test-argocd namespace get below error

Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: ClusterRole "argocd-application-controller" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "test-argocd": current value is "argocd"

**Comment by lchastel** [comment]
According to https://github.com/argoproj/argo-helm/tree/main/charts/argo-cd#synchronizing-changes-from-original-repository, helm chart is using [manifests/install.yaml](https://github.com/argoproj/argo-cd/blob/master/manifests/install.yaml)  not the namespaced one.
